src/clr-angular/utils/focus-trap/focus-trap.directive.ts

```diff
@@ -49,7 +49,8 @@ export class FocusTrapDirective implements AfterViewInit, OnDestroy {
   }
 
   private createFocusableOffScreenEl(): any {
-    const offScreenSpan = this.renderer.createElement('span');
+    // Not using Renderer2's createElement method because that leads to DOM leakage.
```

Comment:
I already approved, but actually can you add here a link to the Angular issue you're about to open. It'll be good to go back to the renderer once they fix the memory leak on their side.

Reply:
Added the URL to the GitHub issue.